### PR TITLE
RIA-2767_add_IAC_API

### PR DESCRIPTION
**Pull request by Lee365**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-2767

### Change description ###
RIA-2767-IAC add documents and notifications

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: docs/specs/ia-case-documents-api.json [hunk 1]
```diff
@@ -0,0 +1,304 @@
+{
```
Comment (timja) [inline docs/specs/ia-case-documents-api.json:1]: this should be published automatically, not through a PR, see https://github.com/hmcts/reform-scan-notification-service/blob/master/.github/workflows/swagger.yml

File: docs/microservices.json [hunk 2]
```diff
@@ -1990,6 +1990,72 @@
...
+              "hard": true,
+              "apis": []
+          },
+          {
+              "id": "cmc-pdf-service",
+              "hard": true,
+              "apis": []
+          },
+          {
+              "id": "idam",
```
Comment (timja) [inline docs/microservices.json:2018]: i think this should be idam-api
```suggestion
              "id": "idam-api",
```

File: docs/specs/ia-case-documents-api.json (hunk 1 above)
Comment (Lee365) [reply docs/specs/ia-case-documents-api.json]: Removed ia-case-documents-api.json and ia-case-notifications-api.json

File: docs/microservices.json (hunk 2 above)
Comment (Lee365) [reply docs/microservices.json]: Corrected to "idam-api" for ia-case-api, ia-case-documents-api and ia-case-notifications-api 
as this is providing links to microservices and "idam" isn’t a microservice.

**Pull request merged**